packages/typegpu/src/data/index.ts

```diff
@@ -136,6 +136,7 @@ export {
   BuiltinNumWorkgroups,
   AnyBuiltin,
 } from '../builtin';
+export type { Exotic as Base } from './exotic';
```

Comment:
will this aliased export also change how Exotic appears as part of other types when hovering over them?

Reply:
Honestly not sure. I did not want to pack too much as part of this PR, we can rename all exotics to base in a separate one.